Modules/_json.c

```diff
@@ -1866,25 +1812,54 @@ PyDoc_STRVAR(module_doc,
 static int
 _json_exec(PyObject *module)
 {
-    if (PyType_Ready(&PyScannerType) < 0) {
+    PyObject *PyScannerType = PyType_FromSpec(&PyScannerType_spec);
```

Comment:
I don't think that it's worth it to have a temporary variable: just writ into `state->PyScannerType` directly.